api/src/repositories/code-repository.ts

```diff
@@ -455,4 +456,27 @@ export class CodeRepository extends BaseRepository {
 
     return response.rows;
   }
+
+  /**
+   * Fetch method response metrics
+   *
+   * @return {ICodeWithDescription}
+   * @memberof CodeRepository
+   */
+  async getMethodResponseMetrics() {
```

Comment:
Needs typescript types

Reply:
```
async getMethodResponseMetrics(): Promise<ICodeWithDescription> {
...
}
```

Reply:
I think this is resolved by adding the Zod schema

Reply:
Often its desirable to specify the return type at the function level as well, so that if someone were to inadvertently alter the `return`, the function would complain.

The `return` is the current return type, but not necessarily the intended return type.

Reply:
I'll add this